data_uploader/app/views/data_uploader/admin/_form_upload_datasets.html.erb

```diff
@@ -48,7 +50,7 @@
       <tbody>
         <% logs.each do |log| %>
           <tr>
-            <td><%= log.created_at %>
+            <td><%= log.created_at.in_time_zone("Eastern Time (US & Canada)") %> (EST)
```

Comment:
the `(EST)` should go between the `<%= and %>` to appear, so:
`<td><%= log.created_at.in_time_zone("Eastern Time (US & Canada)") (EST) %>`

Reply:
The `(EST)` is not part of the code for the timezone, was more for the user information, so that they know that it's in that timezone.